libs/blocks/global-navigation/utilities/menu/menu.js

```diff
@@ -267,6 +267,28 @@ const decorateColumns = async ({ content, separatorTagName = 'H5' } = {}) => {
   }
 };
 
+const decorateLinksStrip = (content) => {
+  const linksStripEl = content.querySelector('.links-strip');
+  if (!linksStripEl) return;
+  linksStripEl.parentElement.removeChild(linksStripEl);
+  const decorateLink = (el) => {
```

Comment:
Since this resembles the markup of link groups, would it not be possible to reuse `decorateLinkGroup`?

Reply:
Yes, we can reuse `decorateLinkGroup`. I modified the code